docker-build.sh

```diff
@@ -8,7 +8,14 @@ DOCKERFILE="${DOCKERFILE:-Dockerfile}"
 cp "$DOCKERFILE" ./build/
 docker build -t "$DOCKER_IMAGE:$TARGET-$BRANCH" -f "./build/$DOCKERFILE" ./build
 
+if [ -n "$ARCH" ]; then
+    docker tag "$DOCKER_IMAGE:$TARGET-$BRANCH" "$DOCKER_IMAGE:$ARCH-$BRANCH"
+fi
+
 if [ "$BRANCH" == "master" ]; then
+    if [ -n "$ARCH" ]; then
+        docker tag "$DOCKER_IMAGE:$TARGET-$BRANCH" "$DOCKER_IMAGE:$ARCH"
+    fi
     docker tag "$DOCKER_IMAGE:$TARGET-$BRANCH" "$DOCKER_IMAGE:$TARGET"
```

Comment:
Shouldn't this be in the else branch? You'll now still create the tag with the TARGET even if the ARCH is set.

Reply:
Arch is only optional. The IB and rootfs share the script and I don't think it's helpful for the IB